src/test/java/seedu/address/logic/parser/book/DeleteBookCommandParserTest.java

```diff
@@ -21,5 +21,8 @@ public void parse_validArgs_returnsDeleteCommand() {
     public void parse_invalidArgs_throwsParseException() {
         assertParseFailure(parser, "a", String.format(MESSAGE_INVALID_COMMAND_FORMAT,
                 DeleteBookCommand.MESSAGE_USAGE));
+
+        assertParseFailure(parser, "-999", String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                DeleteBookCommand.MESSAGE_USAGE));
```

Comment:
I like the crafting of the edge case here and the attention to detail on ensuring that the classes work as intended! 👍